pambase-selinux/PKGBUILD

```diff
@@ -14,7 +14,7 @@ arch=('any')
 url="https://www.archlinux.org"
 license=('GPL')
 groups=('selinux')
-conflicts=("${pkgname/-selinux}" "selinux-${pkgname/-selinux}")
+conflicts=("${pkgname/-selinux}" "selinux-${pkgname/-selinux}" 'pam-selinux>=1.4.0')
```

Comment:
Why is the comparison this way? I would expect `conflicts=(... 'pam-selinux<1.4.0')` instead. Did you test your package and did you succeed in installing both the latest versions of `pam-selinux` and `pambase-selinux` with this patch? If it worked, what am I missing about this `conflicts` comparison?